flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/sink/StoreSinkWrite.java

```diff
@@ -39,6 +40,14 @@ interface StoreSinkWrite {
 
     void compact(BinaryRowData partition, int bucket, boolean fullCompaction) throws Exception;
 
+    void compact(
```

Comment:
Can we just add a `addNewFiles(snapshotId, files)` method here?

Reply:
`notifyNewFiles`